Components/Css/cart.css

```diff
@@ -0,0 +1,54 @@
+.cart-container {
+    display: flex;
+    justify-content: center;
+    margin-top: 3rem;
```

Comment:
This can be also done by creating different utility classes which you can reuse
one for display : flex as d-flex and one for justify-content-center